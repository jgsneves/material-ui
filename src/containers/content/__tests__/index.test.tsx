import { render } from '@testing-library/react';
import ContentContainer from '..';

describe('Content Container test suite', () => {
  it('should render proper snapshot', () => {
    const { asFragment } = render(
      <ContentContainer>
        <h3>ol√°!</h3>
      </ContentContainer>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
