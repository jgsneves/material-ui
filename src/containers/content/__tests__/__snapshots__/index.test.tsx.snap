// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User component test suite should render proper snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-syv4d2-MuiStack-root"
  >
    <h3>
      ol√°!
    </h3>
  </div>
</DocumentFragment>
`;
